doc/installation.rst

```diff
@@ -257,54 +314,76 @@ You'll first want to create and activate a virtual environment in which to insta
 
 With ``virtualenvwrapper``:
 
-.. code-block:: bash
+.. code-block:: shell
 
    mkvirtualenv spyder-env
    workon spyder-env
 
 Otherwise, on macOS/Linux/Unix:
 
-.. code-block:: bash
+.. code-block:: shell
 
    python3 -m venv spyder-env
    source spyder-env/bin/activate
 
 or on Windows:
 
-.. code-block:: bash
+.. code-block:: batch
 
    python -m venv spyder-env
-   spyder-env/Scripts/activate.bat
+   spyder-env\Scripts\activate.bat
 
-After activating your environment, to install Spyder and its other dependencies, run ``pip install spyder``.
+After activating your environment, to install Spyder and its optional dependencies, run:
+
+.. code-block:: shell
+
+   pip install spyder numpy scipy pandas matplotlib sympy cython
+
+Or for a minimal installation, run:
+
+.. code-block:: shell
+
+   pip install spyder
 
 .. image:: /images/installation/installation-pip-install.gif
    :alt: Running Spyder installation with pip
 
-You may need to install a Qt binding (PyQt5) separately with ``pip`` if running under Python 2.
+To launch Spyder after installing, ensure your environment is activated and run the ``spyder`` or ``spyder3`` command.
 
-To launch Spyder after installing, ensure your environment is activated and run the ``spyder3`` command.
 
 
+.. _install-updating:
 
 ===============
 Updating Spyder
 ===============
 
-To update Spyder installed via our :ref:`standalone packages<standalone_installers_ref>` on Windows and macOS, you'll currently need to manually download and install the latest release (if on Windows, make sure to remove the old version first from Control Panel/System Settings).
+By default, Spyder checks for updates automatically on startup.
+You can also do manually with :menuselection:`Help --> Check for updates` on Windows and Linux, or :menuselection:`Spyder --> Check for updates` on macOS.
+
+The :ref:`install-standalone` for Spyder 5.4.0+ also include update functionality built right into Spyder, which after checking for updates as above will display a prompt to automatically download and install the current version.
```

Comment:
@mrclary Is this true on macOS as well? Or did we not add that support in favor of just waiting for the `conda`- based installers? I can't remember the final outcome, sorry...

Reply:
The menu selection on macOS is `Help --> Check for updates`, same as Windows and Linux (should it be different?).

And, yes, the update mechanism works on macOS. The only substantive difference is that macOS users will have to manually drag-and-drop the application.

Reply:
Oh, I thought the canonical location for update checks on macOS was the *<Application>* menu? Partially for that reason, I always thought it was really odd that it Windows applications typically put it under the `Help` menu of all places when I switched to Windows back when I was younger.

Anyway, I've applied a suggestion to update this.